apps/fees-pay/refund-notifications-job/image-policy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImagePolicy
+metadata:
+  name: refund-notifications-app
```

Comment:
Is this correct ? `refund-notifications-app`

Reply:
It has been corrected to 'refund-notifications-job'